src/components/TabsPage/TabsPage.tsx

```diff
@@ -0,0 +1,48 @@
+import { FC } from 'react';
+import {
+  Link,
+  useParams,
+} from 'react-router-dom';
+import cn from 'classnames';
+import { Tab } from '../../types/Tab';
+
+interface Props {
+  tabs: Tab[],
+}
+
+export const TabsPage: FC<Props> = ({ tabs }) => {
+  const { tabId: selectedTabId } = useParams();
+
+  const selectedTabContent = tabs
+    .find(tab => tab.id === selectedTabId)?.content || null;
+
+  return (
+    <>
+      <h1 className="title">Tabs page</h1>
+
+      <div className="tabs is-boxed">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                data-cy="Tab"
+                className={cn({
+                  'is-active': tab.id === selectedTabId,
+                })}
+                key={tab.id}
+              >
+                <Link to={`../${id}`}>{title}</Link>
+              </li>
+            );
+          })}
+        </ul>
+      </div>
```

Comment:
You could split this Page into smaller Components.
For example, this part can be `Tabs` component and `<li>` element can be `TabItem` or `TabLink`.

Also, you may create a separate folder `pages`, and move all pages (`HomePage, NotFoundPage, TabsPage`) there, in `components` leave only smaller parts.